framework/components/ecl-breadcrumbs/ecl-breadcrumbs.twig

```diff
@@ -1,13 +1,12 @@
-<nav class="breadcrumb{{ " breadcrumb--" + modifier|length ? modifier : "" }}" role="navigation" aria-label="breadcrumbs">
+<nav class="ecl-breadcrumb{{ " breadcrumb--" + modifier|length ? modifier : "" }}" aria-label="breadcrumbs">
```

Comment:
Something wrong here. It will output `ecl-breadcrumb breadcrumb--` if `modifier` is empty, or `ecl-breadcrumb breadcrumb--{{modifier}}`. In both cases, the `ecl` prefix is missing